src/map/dto/read-map.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { ResponseSuccessDto } from '../../common/dto/response-success.dto';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class ReadMapDto {
+  @ApiProperty()
+  name: string;
+  address: string;
+  image: string;
+  distance: string;
+}
+
+export class EachRecipeResponseDto extends ResponseSuccessDto {
```

Comment:
```suggestion
export class ResponseMapDto extends ResponseSuccessDto {
```

가 어떨까요?
Each 는 안붙여도 복수/단수형으로 구별하는게 깔끔할 것 같습니다. 
또한 DTO convention 은 nest 공식문서대로 행동과 관련된 부분(ex. create, update, delete, response) 이 앞으로 나오는게 맞을 것 같아요.

Recipe 은 어떤 이유로 이름 붙이신건가요?? Map 이 더 직관적일 것 같아요 
아니면 `ResponseSmokingAreaDto` 라던지

Reply:
청춘식탁에서 베꼈나봐요 ㅋㅋㅋㅋ ㅜ 돌았나? 수정하겠습니다 .. 죄송요 ..

Reply:
디질랜드